src/OidcClient.php

```diff
@@ -44,13 +49,18 @@ public function __construct(
       private string $clientSecret,
       private string $redirectRoute,
       private string $rememberMeParameter,
-      protected ?OidcWellKnownParserInterface $wellKnownParser = null)
-  {
+      private ?string $codeChallengeMethod,
+      protected ?OidcWellKnownParserInterface $wellKnownParser = null,
+  ) {
```

Comment:
For BC reasons we should add new parameters at the end, and provide a default value.

```suggestion
      protected ?OidcWellKnownParserInterface $wellKnownParser = null,
      private ?string $codeChallengeMethod = null)
  {
```